plans/restore.pp

```diff
@@ -12,7 +12,7 @@
   Peadm::SingleTargetSpec $targets,
 
   # restore type determines the restore options
-  Enum['recovery', 'recovery-db', 'custom'] $restore_type = 'recovery',
+  Enum['recovery', 'recovery-db', 'custom', 'migration'] $restore_type = 'recovery',
```

Comment:
I wonder should we also set rbac to false in peadm::migration_opts_default for now - just to reduce errors?